blockchain/difficulty.go

```diff
@@ -244,7 +244,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 			// Return minimum difficulty when more than the desired
 			// amount of time has elapsed without mining a block.
 			reductionTime := b.chainParams.MinDiffReductionTime
```

Comment:
Fair point.  I can change the others.  I mainly didn't do it because they are defined in terms of `time.Duration` which is nanosecond precision, so I was trying to touch as little as possible.  That is also why there is:

```Go
actualTimespan *= int64(time.Nanosecond)`
```